journal/modernized/35-chapter-32.adoc

```diff
@@ -713,10 +714,10 @@ I was not satisfied, however, without making some further attempt to get among t
 and Isaackee was dispatched again, to say, that our coming on shore would be useless,
 without the missionary to interpret.
 He did not at all hesitate to go; but, apprehending some mischief might arise, said,
-on leaving us, '`if they kill me,
-it will be nothing.`' After a considerable parley had taken place, our man returned again,
+on leaving us, "`if they kill me,
+it will be nothing.`" After a considerable parley had taken place, our man returned again,
 and said,
-that '`we might all come on shore;`' but now James Watkin refused to accompany us,
+that "`we might all come on shore;`" but now James Watkin refused to accompany us,
```

Comment:
I think let's either flip this back to single quotes, or maybe eliminate the quotes altogether. The reason I say this is that it's not a direct quotation, the _person_ is wrong.  I find these hard to detect and they confuzzle my brain, so when I'm unsure, I find it helpful to do a little trick, I imagine like I'm watching a play, and I put the words into the actor's mouth and see if it makes sense. So in this case, I imagine a man coming up to Daniel Wheeler and saying "we might all come ashore."  That doesn't make sense, he probably said "You may all come ashore" and then when Wheeler is recounting it, he's saying it informally.

So, after pondering a bit, I think that I lean in this case towards just removing the quotes altogether here in the modernized, and probably leaving it as is in the original.